tests/unittests/tests-crypto/tests-crypto-modes-ccm.c

```diff
@@ -77,21 +83,29 @@ static uint8_t TEST_2_EXPECTED[] = {
     0xCC, 0x15, 0xC4, 0x39, 0xC9, 0xE4, 0x3A, 0x3B,
     0xA0, 0x91, 0xD5, 0x6E, 0x10, 0x40, 0x09, 0x16
 };
-static uint8_t TEST_2_EXPECTED_LEN = 40;
-
-static void test_encrypt_op(uint8_t* key, uint8_t key_len, uint8_t* adata,
-                            uint8_t adata_len, uint8_t* nonce, uint8_t nonce_len, uint8_t* plain,
-                            uint8_t plain_len, uint8_t* output_expected, uint8_t output_expected_len)
+static size_t  TEST_2_EXPECTED_LEN = 40;
```

Comment:
nit: extra space after `size_t`

Reply:
I added it on purpose to align with the other variables definitions, is that bad ?

Reply:
we typically don't do that, at least not after the type, but rather for the value. Hence please remove extra space